CIP-0116/cardano-babbage.json

```diff
@@ -959,6 +959,7 @@
         },
         "withdrawals": {
           "type": "array",
+          "minItems": 1,
```

Comment:
Whats the rationale for this?

Reply:
Hmmm, my mistake

https://github.com/IntersectMBO/cardano-ledger/blob/a8b02a7cd32ffb453c95805450440b6bd4dc7014/eras/babbage/impl/cddl-files/babbage.cddl#L261

https://github.com/IntersectMBO/cardano-ledger/blob/a8b02a7cd32ffb453c95805450440b6bd4dc7014/eras/conway/impl/cddl-files/conway.cddl#L506

It only changed in conway